README.md

```diff
@@ -57,6 +57,15 @@ Each release of Dapr CLI includes various OSes and architectures. These binary v
    * For Linux/MacOS - `/usr/local/bin`
    * For Windows, create a directory and add this to your System PATH. For example create a directory called `c:\dapr` and add this directory to your path, by editing your system environment variable.
 
+#### From the MSI Package (windows)
+
+Install windows Dapr CLI using MSI package.
+
+1. Download the [MSI Package](https://github.com/dapr/cli/releases)
+2. Run `dapr.msi` to install Dapr CLI on your system
```

Comment:
Do we need to run this command in administrator mode or not required ?

Reply:
No administrator mode is not required.